pages/settings/SettingsLayout.qml

```diff
@@ -290,34 +308,30 @@ Rectangle {
 
     ListModel {
         id: fiatPriceCurrencyModel
-        ListElement {
-            data: "xmrusd"
-            column1: "USD"
-        }
-        ListElement {
-            data: "xmreur"
-            column1: "EUR"
+        Component.onCompleted: {
+            // populate with fiat currencies
+            appWindow.fiatCurrencies.forEach(el => {
+                fiatPriceCurrencyModel.append({ data: `xmr${el}`, column1: el.toUpperCase()});
+            });
         }
     }
 
     Component.onCompleted: {
-        // Dynamically fill fiatPrice dropdown based on `appWindow.fiatPriceAPIs`
+        // Dynamically fill fiatPrice dropdowns based on `appWindow.fiatPriceAPIs`
         var apis = appWindow.fiatPriceAPIs;
         fiatPriceProvidersModel.clear();
-
         var i = 0;
-        for (var api in apis){
-            if (!apis.hasOwnProperty(api))
+        for (var apiProvider in apis){
+            if (!apis.hasOwnProperty(apiProvider))
                continue;
 
-            fiatPriceProvidersModel.append({"column1": Utils.capitalize(api), "data": api});
+            fiatPriceProvidersModel.append({data: apiProvider, column1: Utils.capitalize(apiProvider)});
 
-            if(api === persistentSettings.fiatPriceProvider)
+            if(apiProvider === persistentSettings.fiatPriceProvider)
                 fiatPriceProviderDropDown.currentIndex = i;
             i += 1;
         }
 
         console.log('SettingsLayout loaded');
     }
 }
-
```

Comment:
revert all the unnecessary white space changes to reduce the diffs of this PR please

Reply:
Just let it be.